packages/components/src/styles/container/container.declaration.ts

```diff
@@ -21,7 +21,7 @@ export default {
       name: 'sd-container--triangle-...',
       options: ['top', 'right', 'bottom', 'left'],
       description:
-        'Defines an optional triangle cut-out for sd-container. This allows for an indentation resembling an arrow on any side of the container. CSS Property `triangle-background` defines the background color of the cut-out.'
+        'Defines optional triangle cut-out for sd-container. This allows for an indentation resembling an arrow on any side of the container. CSS Property `triangle-background` defines the background color of the cut-out.'
```

Comment:
Very minor but I think it makes sense to keep the "an". Approving since this is non-blocking.

```suggestion
        'Defines an optional triangle cut-out for sd-container. This allows for an indentation resembling an arrow on any side of the container. CSS Property `triangle-background` defines the background color of the cut-out.'
```

Reply:
Haha, I agree, I just needed the pipeline to run and for some reason merging main did not work...